src/components/Header/HeaderContact.js

```diff
@@ -17,10 +18,27 @@ const HeaderContact = ({
   disabledNextBtn
 }) => {
   const renderHeader = () => {
+    if (Platform.OS === 'android') {
+      return (
+        <GlobalButton testID="onPressAndroid" buttonStyle={styles.noPaddingLeft} onPress={onPress}>
+          <View testID="android" style={styles.content(-4)}>
+            <ArrowLeftIcon
+              width={dimen.normalizeDimen(20)}
+              height={dimen.normalizeDimen(20)}
+              fill="#000"
+            />
+          </View>
+        </GlobalButton>
+      );
+    }
     return (
-      <GlobalButton testID="onPressBtnHeader" buttonStyle={styles.noPaddingLeft} onPress={onPress}>
-        <View style={styles.content}>
-          <ArrowLeftIcon width={20} height={20} fill="#000" />
+      <GlobalButton testID="onPressIos" buttonStyle={styles.noPaddingLeft} onPress={onPress}>
+        <View testID="ios" style={styles.content(-8)}>
+          <ArrowLeftIcon
+            width={dimen.normalizeDimen(20)}
+            height={dimen.normalizeDimen(20)}
+            fill="#000"
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `renderHeader` function has been updated to conditionally render different components based on the platform (Android or iOS). This is a good practice as it allows for platform-specific customization. However, the only difference between the two platforms seems to be the marginLeft style property. It might be more efficient to determine the marginLeft value based on the platform and then use that in a single return statement, reducing code duplication.
```diff
-    return (
-      <GlobalButton testID="onPressBtnHeader" buttonStyle={styles.noPaddingLeft} onPress={onPress}>
-        <View style={styles.content}>
-          <ArrowLeftIcon width={20} height={20} fill="#000" />
-         </View>
-       </GlobalButton>
-     );
+    const marginLeft = Platform.OS === 'android' ? -4 : -8;
+    return (
+      <GlobalButton testID={`onPress${Platform.OS}`} buttonStyle={styles.noPaddingLeft} onPress={onPress}>
+        <View testID={Platform.OS} style={styles.content(marginLeft)}>
+          <ArrowLeftIcon
+            width={dimen.normalizeDimen(20)}
+            height={dimen.normalizeDimen(20)}
+            fill="#000"
+          />
+        </View>
+      </GlobalButton>
+    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->